pkg/ostree/index.html

```diff
@@ -26,8 +26,13 @@
   <link href="ostree.css" rel="stylesheet">
   <script src="../base1/bundle.js"></script>
   <script src="ostree.js"></script>
+
+  <!-- debug:start -->
+  <script src="app.js"></script>
```

Comment:
Should we load client.js here too. To make sure the module is fully loaded by the time we get to ng-app.